server/models/Notification.js

```diff
@@ -132,6 +132,28 @@ export default function(Sequelize, DataTypes) {
       },
     }).then(us => us.map(us => us.UserId));
   };
+  /**
+   * Check if notification with `notificationType`, `CollectiveId` and `UserId` is active.
+   * @param {*} notificationType
+   * @param {*} CollectiveId
+   * @param {*} UserId
+   */
+  Notification.isActive = (notificationType, CollectiveId, UserId) => {
+    debug('isActive', notificationType, CollectiveId, UserId);
+    return models.Notification.findOne({
+      where: {
+        CollectiveId,
+        type: notificationType,
+        UserId,
```

Comment:
I don't think `UserId` is necessary here, `CollectiveId` is enough.

Reply:
I think it is necessary to identify if the recipient of the email wants it, remember it's a "thankyou" message to a particular user.

Reply:
I'll double check.

Reply:
Ok, so the un-subscription is hyper granular, it will only unsubscribe from future "thankyou" emails related to a given collective.

Example:  if I get a "Thank you for giving to Babel",  and click the unsubscribe link in the email, I will be unsubscribed from future  "Thank you for giving to Babel", but not "Thank you for giving to Webpack".

In that regard, `UserId` and `CollectiveId` usage is effectively fine.